hw/ip/prim/prim_clock_gp_mux2.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "lowrisc:prim:clock_gp_mux2"
+description: "2-input clock multiplexer"
```

Comment:
sorry one last thing, do you mind in the core file specifying that this is a glitchless multiplexer?

Reply:
NP. I will add it in the core file.